ui/qt-app/Glue.hs

```diff
@@ -162,6 +170,12 @@ knitFaceToUI UiFace{..} KnitFace{..} putPass =
           fromResult result >>= fromValue >>= \case
             Knit.ValueHash h -> Right $ pretty h
             _ -> Left "Unrecognized return value"
+      UiCalcTotal {} ->
+        Just . UiCalcTotalCommandResult . either UiCalcTotalCommandFailure UiCalcTotalCommandSuccess $
+          fromResult result >>= fromValue >>= \case
+            Knit.ValueCoin c ->
+              let (amount, unit) = Knit.showCoin c in Right (amount, show unit)
+            _ -> Left "Total calculation error"
```

Comment:
Maybe let's show something more descriptive? For example attach the value that you ignore using `_`?

Reply:
AFAIU, I can't get any additional info from `Knit.ComponentValue components Knit.Cardano`, so it's proper to change message to `Unrecognized return value`.
Correct me, if I'm wrong

Reply:
> deriving instance Show (Value components) => Show (ComponentValue components Core)

You should be able to `show` it at least.